tests/garage/tf/algos/test_dqn.py

```diff
@@ -55,7 +55,7 @@ def test_dqn_cartpole(self):
                 n_epochs=n_epochs,
                 n_epoch_cycles=n_epoch_cycles,
                 batch_size=sampler_batch_size)
-            assert last_avg_ret > 20
+            assert last_avg_ret > 15
```

Comment:
did you check with @ahtsan about this?

lowering the expected performance of an algorithm is a pretty serious change to a test. we need to make sure the new changes didn't cause the performance to drop.

Reply:
Yes we discussed. We both confirm it is returning ~30 but sometimes ~20 on Travis. I believe it is another flaky test.

Reply:
checked with anson